backend/index.js

```diff
@@ -49,6 +49,33 @@ app.get('/', (req, res) => {
   })
 
 
+  app.post('/exercises/:id/sets', async (req, res) => {
```

Comment:
nit: add try catch to log error. Similar to other async calls.

Reply:
Okay!